helpme/views/helpme.py

```diff
@@ -34,20 +35,20 @@ def form_valid(self, form):
         user_agent = self.request.user_agent
 
         if user_agent.is_mobile:
-            device = "Mobile",
+            device = _("Mobile"),
         elif user_agent.is_pc:
-            device = "PC"
+            device = _("PC")
         elif user_agent.is_tablet:
-            device = "Tablet"
+            device = _("Tablet")
         else:
-            device = "Unknown"
+            device = _("Unknown")
 
         form.instance.user_meta = {
-            "Browser": user_agent.browser.family + " " + user_agent.browser.version_string,
-            "Operating System": user_agent.os.family + " " + user_agent.os.version_string,
-            "Device": user_agent.device.family,
-            "Mobile/Tablet/PC": device,
-            "IP Address": self.request.META['REMOTE_ADDR']
+            _("Browser"): user_agent.browser.family + " " + user_agent.browser.version_string,
```

Comment:
We don't want data keys to be translated and should handle that in the template.  They should be consistent in values since they are used by code.

Reply:
for example this should be
`{'browser': {'family': user_agent.browser.family, 'version': user_agent.browser.version_string...`